README.md

```diff
@@ -264,6 +268,92 @@ Query("Select+Id,Name+From+Account")
 >>> "/query?q=Select+Id,Name+From+Account"
 ```
 
+#### Customize Querystring parameter formatting
+
+Using `RefitSettings.UrlParameterKeyFormatter` and `RefitSettings.UrlParameterFormatter`, you can customize the way querystrings
```

Comment:
This PR only introduces `UrlParameterKeyFormatter` so I did not document existing functionality or an example of `UrlParameterFormatter`